stytra/gui/protocol_control.py

```diff
@@ -35,6 +35,14 @@ class ProtocolControlToolbar(QToolBar):
     """ Emitted via the toggle button click, meant to
                          abort the protocol."""
 
+    sig_start_tracking = pyqtSignal()
```

Comment:
Do you need those signals to leave in the protocol control class? It is not intuitive why to me. I guess you need the signals, but why not defining them from the experiment?